### ISSUE TITLE
Content and IA Request for temporary medical records landing page from mhv-medical-records 

### ISSUE THREAD

**Issue by alexiawunder** [department-of-veterans-affairs/va.gov-team#55512]
## What does your team need support for? Check all that apply.
- [X] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [ ] Something else 
 
As we begin moving into Phase 0, medical records on VA.gov will need a way to house any FAQ questions, explain how to submit feedback, and a way to navigate back to MHV classic. Because Phase 0 will only be accessible via URL, and it won't be connected to the My Health hub landing page yet, we are proposing this temporary landing page. It will serve as a separate sibling page that can be retired or easily moved to resources and support in the future, rather than as a parent over this section. 

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [X] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
October 2024  

## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline: [Medical Records Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records)
- [Link to user flow](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/FF607397-FAC4-43C3-B5CA-32404471EA02)
- Link to prototype or mockups:
     - Temp landing page [mobile](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/CD6EFAE2-8F48-4C0C-A5F3-F068F5A2775B) | [desktop](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/C5CADBEE-C7F8-4FDA-B28D-5B648416EB37)
- Link to any epics/issues that may be helpful: N/A

## Will this work be going through the Collaboration Cycle?
- [X] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: ~June 2023
- Estimated staging review date:~June 2023
- Content and IA work needed by: We'd like to conduct usability testing with approved content at the end of March, 2023

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [X] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: mhv-medical-records
- OCTO-DE product owner: Lauren Alexanderson
- Product manager: Bryan Riley
- Designer: Alexia Wunder, Bobby Bailey
- FE engineer: Mike Moyer
- Product/team Slack channel: #mhv-medical-records

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by laurwill** [comment]
Hi @alexiawunder ,
We have a working draft of this page to use for testing — we'll likely add to and adjust the FAQs section before phase 0 launch, but wanted to get plausible content into the prototype before midpoint. 

Here's the Sharepoint doc with working drafts of all the larger content pieces across domains: [​Medical records content working draft.docx icon Medical records content working draft.docx](https://dvagov-my.sharepoint.com/:w:/g/personal/laura_willwerth_va_gov/Eacz1GNGqrdNn3b5gC1j0n4BjNyZ6LyafJCpEfebp-cPVg?e=f6e3Hy)

Also noting that we weren't sure we needed the links and blurbs for the 3 main sections of the tool here — those will be in the left nav on this screen, right? If you think we need those in the landing page, let's chat and we can quickly add some blurbs for those. Thank you!

cc @AnneCostello @BobbyBaileyRB @aliyahblackmore
